RadixWallet/Profile/Logic/Profile/Profile+AppPreferences+Update.swift

```diff
@@ -11,19 +11,3 @@ extension Profile {
 		self.appPreferences.updateDisplay(display)
 	}
 }
-
-extension Profile {
-	/// Appends a new `P2PLink` to the Profile's `AppPreferences`, returns `nil` if it was not inserted (because already present).
-	@discardableResult
-	public mutating func appendP2PLink(_ p2pLinks: P2PLink) -> P2PLink? {
```

Comment:
if the code compiles without removing this code, please re-add it :) the whole `RadixWallet/Profile` folder is deleted in https://github.com/radixdlt/babylon-wallet-ios/pull/1085

Reply:
It does not compile. We have to re-add `p2pLinks` to `AppPreferences`

Reply:
Right, thanks! Unavoidable merge conflict pain 😅